docs/book.toml

```diff
@@ -0,0 +1,6 @@
+[book]
+authors = ["Sven Lechner", "vringar"]
```

Comment:
maybe we can just make the authors something more generic like `spotifyd contributers`? what do you think @JojiiOfficial / @Spotifyd/maintainers

Reply:
Ye, I am looking into setting the contributors thing up. I already installed the bot but didn't go through the configuration yet. As for now, just set it to something like `spotifyd team & contributors` or even use the official team name `@Spotifyd/maintainers`.